set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/web/WebUserSetPublishingIT.java

```diff
@@ -135,7 +141,7 @@ public void updatePublishedUserSetWithVisibility_Success() throws Exception {
     WebUserSetImpl userSet = createTestUserSet(USER_SET_REGULAR, regularUserToken);
 
     // publish userset by other user, the ownership stays with the creator
-    publishUserSet(userSet, USERNAME_REGULAR);
+    publishUserSet(userSet, null, USERNAME_REGULAR);
```

Comment:
should verify the value of the issues field, the current date take before calling the method can be used for comparison as issues.after(requestTime)